src/js/contentUtils.js

```diff
@@ -484,12 +486,38 @@ export function hasInvalidAttributes(htmlElement) {
       }
     });
   }
-  // check child nodes as well, since a malicious attacker could try to inject an invalid attribute via an image node in a svg tag
+  // check child nodes as well, since a malicious attacker could try to inject an invalid attribute via an image node in a svg tag using a use element
   if (htmlElement.childNodes.length > 0) {
     htmlElement.childNodes.forEach(childNode => {
       if (childNode.nodeType === 1) {
         hasInvalidAttributes(childNode);
       }
+      // if the element is a math element, check all the attributes of the child node to ensure that there are on href or xlink:href attributes with javascript urls
+      if (
+        htmlElement.tagName.toLowerCase() === 'math' &&
+        Object.keys(childNode.attributes).length >= 1
+      ) {
+        Array.from(childNode.attributes).forEach(elementAttribute => {
+          if (
+            (elementAttribute.localName === 'href' ||
+              elementAttribute.localName === 'xlink:href') &&
+            childNode
+              .getAttribute(elementAttribute.localName)
+              .toLowerCase()
+              .includes('javascript')
```

Comment:
does it need to be includes or can it be `startsWith`?

Reply:
yeah, we can make it startsWith